src/main/java/com/p6spy/engine/event/CompoundJdbcEventListener.java

```diff
@@ -39,7 +39,7 @@ public CompoundJdbcEventListener(List<JdbcEventListener> eventListeners) {
     this.eventListeners = eventListeners;
   }
 
-  public void addListender(JdbcEventListener listener) {
```

Comment:
While this is a right thing to do it may be called by our users so it needs to be deprecated first. At least I [did it](https://github.com/gavlyukovskiy/spring-boot-data-source-decorator/blob/e38f23b987dedf386a39023e1e53e5025b5e6886/datasource-decorator-spring-boot-autoconfigure/src/main/java/com/github/gavlyukovskiy/boot/jdbc/decorator/p6spy/ContextJdbcEventListenerFactory.java) :)

Reply:
I added back the old method with the type and marked it @deprecated.